Index.md

```diff
@@ -26,6 +26,8 @@
 | [Custom-Range-Slider](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Custom-Range-Slider)                   | Simple Custom Range Slider written in HTML, CSS, and JavaScript                                                                                                                                                                      |
 | [Digital Clock](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/DIGITAL-JAVASCRIPT-CLOCK)                    | Simple Clock App written in HTML, CSS, and JavaScript                                                                                                                                                                                |
 | [Expense Tracker](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Expense%20Tracker)                         | Keep a track of your own income and expenses and have a clear picture of where your money is going                                                                                                                                   |
+| [FAQ-Accordion]()                                                                                                                | A accordion is a useful way to squeeze lots of page content into a small space
+             |
 | [Google.com Clone](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/GOOGLE.COM%20PAGE%20PROJECT)              | A google clone which also lets you search information, looks exactly same as Google                                                                                                                                                  |

```

Comment:
The view of the index.md has been messed up. Please fix it. Please keep it in the form of a table itself.

Reply:
Done!